assets/js/script.js

```diff
@@ -93,4 +93,12 @@ $(document).ready(function() {
 		$('#warn-invalid-link').attr({ 'href': href, 'target': target });
 		$('#warn-invalid-modal').modal();
 	});
+
+	// Comment this code to debug the site in the console
+    console.clear()
```

Comment:
As suggested by the condition on line 99, `console` may be undefined here.
This line of code is stupid.

```suggestion
```

Reply:
Oh and inconsistent indentation as a bonus.